src/builtins.rs

```diff
@@ -72,6 +73,7 @@ fn string_equals(ctx: &mut Context, args: Args) -> Option<ObjectInstance> {
 }
 
 /// Link with a given library at runtime
+#[cfg(feature = "ffi")]
```

Comment:
I think here it would be better to emit an error and return None so that the user has visual feedback instead of an invalid builtin error or something. Something custom like "jinko was not compiled with ffi support" as you did for the REPL would be nice!

Reply:
Fixed